Source/WebKit/UIProcess/API/glib/WebKitSettings.cpp

```diff
@@ -4131,3 +4131,81 @@ WebKitFeatureList* webkit_settings_get_development_features(void)
 {
     return webkitFeatureListCreate(WebPreferences::internalDebugFeatures());
 }
+
+/**
+ * webkit_settings_apply_from_key_file:
```

Comment:
Do we want this to apply settings at any time to an existing settings, or to create a new settings object with its values loaded from a key file?

Reply:
This API does the former. I suppose the latter could also be a valid use-case but it can be covered using this API already, on application side.